### ISSUE TITLE
My val_psnr and test_psnr are different 

### ISSUE THREAD

**Issue by Fangzhenxuan** [JingyunLiang/MANet#7]
Thanks for sharing your work.
I use the same test set, My val_psnr is 30 dB during traing, but I only got 25dB during testing. Do you know why?

**Comment by JingyunLiang** [comment]
Can you try to test your data with the pretrained model [here](https://github.com/JingyunLiang/MANet/releases)? So that we can know where the problem comes from (data preparation or model training).
